src/main/resources/db/migration/V090__admin_email_categorisation.sql

```diff
@@ -0,0 +1,90 @@
+--- FACT-95 ---
+--- Adding new table for admin_email_description
+CREATE TABLE public.admin_emailtype
+(
+    id              integer PRIMARY KEY    NOT NULL,
+    description     character varying(250) NOT NULL,
+    description_cy  character varying(500) NOT NULL
+);
+
+CREATE SEQUENCE public.admin_emailtype_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+ALTER SEQUENCE public.admin_emailtype_id_seq OWNED BY public.admin_emailtype.id;
+
+ALTER TABLE ONLY public.admin_emailtype
+ALTER COLUMN id SET DEFAULT nextval('public.admin_emailtype_id_seq'::regclass);
+
+-- Amending the existing search_email table to include a foreign key to the one above
+ALTER TABLE public.search_email
+ADD COLUMN admin_email_type_id integer NULL;
+
+-- Inserting the data for the new table
+INSERT INTO public.admin_emailtype(description, description_cy)
+VALUES
+('Case progression', 'Case progression cy'), ('Chancery', 'Chancery cy'),
```

Comment:
Needs to be confirmed before merging in! Dw I won't go all cray and merge it in until then :   -   )